orderbook/src/database.rs

```diff
@@ -57,4 +57,45 @@ impl Database {
             .await?;
         Ok(())
     }
+
+    async fn count_rows_in_table(&self, table: &str) -> Result<i64> {
+        let query = format!("SELECT COUNT(*) FROM {};", table);
+        let row = self.pool.fetch_one(query.as_str()).await?;
+        row.try_get(0).map_err(Into::into)
+    }
+
+    pub async fn count_rows_in_tables(&self) -> Result<HashMap<&'static str, i64>> {
+        let tables = [
+            "orders",
+            "trades",
+            "invalidations",
+            "min_fee_measurements",
+            "settlements",
+        ];
```

Comment:
Can we make Tables an enumerable struct similar to `Liquidity` (EnumVariantNames) so that we don't store our list of "all tables" in multiple places? The truncate queries could use the same list

Reply:
I made it a const list.